Podcast/SeriesGridCollectionViewCell.swift

```diff
@@ -63,16 +64,27 @@ class SeriesGridCollectionViewCell: UICollectionViewCell {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+}
 
-    func configureForSeries(series: Series, showLastUpdatedText: Bool = false) {
-        imageView.setImageAsynchronouslyWithDefaultImage(url: series.largeArtworkImageURL)
-        titleLabel.text = series.title
-        
-        if showLastUpdatedText {
-            subscribersLabel.text = series.lastUpdatedString == "" ? "Never updated" : "Last updated \(series.lastUpdatedString)"
+extension SeriesGridCollectionViewCell: SeriesDisplayView {
+    func set(title: String) {
+        titleLabel.text = title
+    }
 
-        } else {
-            subscribersLabel.text = series.numberOfSubscribers.shortString() + (series.numberOfSubscribers == 1 ? " Subscriber" : " Subscribers")
-        }
+    func set(lastUpdated: String) {
+        subscribersLabel.text = lastUpdated == "" ? "Never updated" : "Last updated \(lastUpdated)"
     }
+
+    func set(numberOfSubscribers: Int, isSubscribed: Bool) {
+        subscribersLabel.text = numberOfSubscribers.shortString() + (numberOfSubscribers == 1 ? " Subscriber" : " Subscribers")
+    }
+
+    func set(largeImageUrl: URL) {
+        imageView.setImageAsynchronouslyWithDefaultImage(url: largeImageUrl)
+    }
+
+    func set(smallImageUrl: URL) {}
+    func hideSubscribedButton() {}
+    func set(author: String) {}
+    func set(topicsString: String) {}
```

Comment:
I think the code is well-organized and I like how you guys are trying to centralize you cell set-up code. One thing that I don't like is how there's a lot of empty, unimplemented methods with this protocol. I feel like the protcol `SeriesDisplayView` in this example (`SeriesGridCollectionViewCell`) should describe exactly what methods one needs to "set" to set up the `SeriesGridCollectionViewCell`. B/c otherwise someone can just call `set(smallImageUrl: URL)` on a `SeriesGridCollectionViewCell` but that does nothing to a `SeriesGridCollectionViewCell` and therefore the code shouldn't allow `set(smallImageUrl: URL)` to be called since that's misleading. 

As to a solution, I'm still wrangling w it myself. I guess you could make the methods that not all cells implement optional. Alternatively you could have a base protocol that has methods that all cells implement and have more specific protocols inherit from that base protocol. And maybe use the visitor pattern to distinguish between different protocol types. At the end of the day, I feel the code should write in such a way that it is unambiguous on exactly what setter methods need to be called in `cellForRowAt` and the codes shouldn't allow the calling of setter methods that have empty implementations from every cell's `displayView`